app/controllers/reviews_controller.rb

```diff
@@ -10,6 +10,10 @@ def create
     @review.food_truck_id = @food_truck.id
     @review.user = current_user
     if @review.save
+      unless @food_truck.user == nil
+        UserMailer.review_notification(
+        @food_truck.user, current_user, @food_truck).deliver_now
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.